ontotext-yasgui-web-component/src/models/external-yasgui-configuration.ts

```diff
@@ -31,6 +31,11 @@ export interface ExternalYasguiConfiguration {
    */
   showResultTabs: boolean;
 
+  /**
+   * If the result info should be rendered or not
```

Comment:
What is this resultInfo which should be rendered or not. Maybe more detailed explanation would be better.

Reply:
Done